### DISCUSSION TITLE
Error building example apps with use_frameworks! and custom build configurations

### DISCUSSION THREAD

**Discussion by dcacenabes** [swiftyfinch/Rugby#380] [Q&A] [answered]
Hi,

We have a project that uses custom build configurations (instead of default Debug and Release). When using Rugby and trying to build one of the example apps that the project has, the modulename-ExampleApp-frameworks.sh script fails with the following error:
``` 
/Users/cace/Developer/test/RugbyIssueDemoUseFrameworks/Pods/Target Support Files/LocalPod/LocalPod-ExampleApp-frameworks.sh: line 42: source: unbound variable
```
The issue only happens with `use_frameworks!` option enabled in the Podfile

Steps to reproduce:

- Download example project [RugbyIssueDemoUseFrameworks.zip](https://github.com/swiftyfinch/Rugby/files/14665183/RugbyIssueDemoUseFrameworks.zip).
- Run:
  - `bundle install`
  - `bundle exec pod install`
  - `rugby --config DebugStaging`
  - `xed .`
- Select LocalPod-ExampleApp scheme
- Build

Expected output:
App builds successfully

Actual output:
App fails

**Comment by dcacenabes** [comment]
[RugbyIssueDemoUseFrameworks.zip](https://github.com/swiftyfinch/Rugby/files/14670278/RugbyIssueDemoUseFrameworks.zip)
The previous example project contains an error that prevents it from building, this one is updated

**Comment by swiftyfinch** [comment]
Hello @dcacenabes!

- I've added a fix here: #381 

Thanks a lot for your detailed description and an example 👍
It's helped me a lot.

**Reply by swiftyfinch** [reply, marked as answer]
🏈 [**v2.7.1**](https://github.com/swiftyfinch/Rugby/releases/tag/2.7.1)

**Comment by dcacenabes** [comment]
Thanks a lot to you! Really appreciate the fast response here :)

Tested it with our main project and now I fully can build it caching everything 🎉 

I might open a separate issue because we're dealing with one dependency that causes the build to fail. Of course I can always exclude that dependency as a workaround, but it'd be great to understand why it fails and if it can be fixed in a generalist way.

**Reply by swiftyfinch** [reply]
Nice to hear that! 👍
<br>

> I might open a separate issue because...

Yeah, feel free to do so.
